homeassistant/components/mutesync/translations/en.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This file is out of sync with `string.json`.

Run `python -m script.translations develop` from the project root to update it.